src/modm/platform/spi/sam/module.lb

```diff
@@ -14,7 +14,11 @@ def get_properties(env):
     device = env[":target"]
     driver = device.get_driver("spi")
     properties = {}
-    properties["use_fiber"] = env.get(":processing:protothread:use_fiber", False)
+
+    use_fiber = env.get(":processing:protothread:use_fiber", False) or \
+        (env.has_module(":processing:fiber") and not env.has_module(":processing:protothread"))
```

Comment:
Yeah, good point. You could but this into a environment query in the fiber module (maybe `:processing:fiber:__enabled`, leading double underscore hides it during discovery) and then only define this logic there there once.

Reply:
Thanks for the suggestion. I'll look into it.